src/iab.ts

```diff
@@ -141,23 +176,8 @@ export const TcfGvlV3Vendor = t.intersection([
     usesCookies: t.boolean,
     cookieRefresh: t.boolean,
     usesNonCookieAccess: t.boolean,
-    dataRetention: t.intersection([
-      t.type({
-        purposes: t.record(t.string, t.number),
-        specialPurposes: t.record(t.string, t.number),
-      }),
-      t.partial({
-        stdRetention: t.number,
-      }),
-    ]),
-    urls: t.array(
-      t.intersection([
-        t.type({ langId: t.string, privacy: t.string }),
-        t.partial({
-          legIntClaim: t.string,
-        }),
-      ]),
-    ),
+    dataRetention: TcfGvlV3DataRetention,
+    urls: TcfGvlV3Urls,
```

Comment:
```suggestion
    urls: t.array(TcfGvlV3Url),
```
following the above suggestion.

Reply:
Ahhh it much more flexible now! I made the change and it should be ready for re-review! Thanks!